prowler/providers/aws/services/eks/eks_service.py

```diff
@@ -101,6 +102,7 @@ class EKSCluster(BaseModel):
     name: str
     arn: str
     region: str
+    version: str = None
```

Comment:
```suggestion
    version: str
```

Reply:
I have tried this before, it was causing other unit tests to fail for other services.